### PR TITLE
Configure testing

### PR DESCRIPTION
**Pull request by fitzsimonsdevin**
# Summary of changes
- Setup Enzyme configuration
- Establish that CSS is ready so work can begin in earnest

# Acceptance Criteria
- [x] My code follows the style guidelines of this project
- [ ] I have run eslint and prettier and have no errors in my files
- [x] I have performed a self-review of my own code
- [ ] CodeClimate has no errors. If there are CodeClimate errors that you choose not to fix, please note why below and initiate a discussion with the team concerning the validity of the errors.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import './index.css'
```
Comment (kyleruschel) [inline src/index.js]: We use sass, so I would use sass in this tutorial as well. So install the required dependencies and update the extensions and I will approve 👍

Comment (fitzsimonsdevin) [reply src/index.js]: Done

File: package.json [hunk 2]
```diff
@@ -7,19 +7,26 @@
     "npm": "6.4.1"
   },
   "dependencies": {
+    "node-sass": "^4.12.0",
```
Comment (kyleruschel) [inline package.json]: I think `node-sass` for this may be unnecessary, but it keeps you in line with the client so all good 👍

Comment (fitzsimonsdevin) [reply package.json]: It def isn't needed. It's an artifact from when I tried to add Sass without updating `create-react-app` versions. I'll remove then merge in.

**Pull request merged**